app/models/concerns/digital_object/datacite.rb

```diff
@@ -48,7 +33,7 @@ def mint_and_store_doi(identifier_status, target_url = nil)
   # returns true if status change was successful
```

Comment:
Does this method ever return false anymore?  Or does the comment need to be updated?

Reply:
Comment updated as suggested.